flambe/field/text.py

```diff
@@ -85,6 +86,9 @@ def __init__(self,  # nosec
             Whether to drop tokens that don't have embeddings
             associated. Defaults to True.
             Important: this flag will only work when using embeddings.
+        setup_all_embeddings: bool
```

Comment:
I don't think it's super clear what happens from this description. Maybe say that all words from the optional provided embeddings will be added to the vocabulary and to the embedding matrix

Reply:
Good point - will use that example